test/unit_tests/braket/default_simulator/openqasm/test_interpreter.py

```diff
@@ -1636,7 +1636,6 @@ def test_noise():
         "int x = 1 + 1;",
         "qubit[2] q; h q[0:1];",
         "gate my_x q { x q; }",
-        "qubit q; gphase(.3);",
```

Comment:
why did we remove this?

Reply:
Because I removed the warning for the `gphase` operation, so this test case is no longer valid. We don't need that warning, since it's listed as a supported operation in the device capabilities and different supported operations don't typically get any special treatment via warnings etc